dashboard/test/controllers/registrations_controller/demigrate_from_multi_auth_test.rb

```diff
@@ -22,6 +22,7 @@ class DemigrateFromMultiAuthTest < ActionDispatch::IntegrationTest
 
   test "demigrate_from_multi_auth is a no-op for an unmigrated user" do
     teacher = create :teacher
+    teacher.demigrate_from_multi_auth
     refute teacher.migrated?
```

Comment:
Looks like this test no longer tests what it says it's testing.  Use `:demigrated` trait _and_ call `demigrate_from_multi_auth`?